### PR TITLE
Thumbnail screenshots on the frontpage

### LINKED ISSUE #706: Compress images on the front page
E.g. https://delta.chat/assets/blog/screenshots/2019-12-17-delta-chat-google-play-release-group-light.png is >700kb currently.
Would be nice to resize them, name something like `thumbnail...` and link `img` tag to them with a link to the original screenshot on click.

### PR DESCRIPTION
**Pull request by link2xt**
I also added explicit height to image tags to prevent layout shifts.

Closes #706

### CODE REVIEW

**Comment by link2xt** [comment]
@r10s Do we need to push the new page for translation to transifex afterwards or do I need to change all the translated pages as well? For now translations still have large screenshots which is also not that bad.

**Comment by missytake** [comment]
I'm surprised how well this works :D there is one difference to before though.

## New:

Screenshot_2023-08-01_00-13-40

## Old:

Screenshot_2023-08-01_00-13-58

I like the old one slightly better, even though it's not the best option it could be I think.

**Comment by link2xt** [comment]
> there is one difference to before though.

I don't know what causes it and how to restore the old behavior

File: en/index.md [hunk 1]
```diff
@@ -14,14 +14,20 @@ lang: en
...
+<a href="../assets/blog/screenshots/2019-12-17-delta-chat-google-play-release-group-light.png">
+<img src="../assets/blog/screenshots/2019-12-17-delta-chat-google-play-release-group-light-thumbnail.png" width="120" height="213"
+style="float: left; margin: 10px;display: block;box-shadow: 5px 5px 2px #777;" alt="A screenshot of Delta Chat on Android showing a chat" />
+</a> 
+<a href="../assets/blog/desktop-screenshot.png">
+<img src="../assets/blog/desktop-screenshot-thumbnail.png" width="280" height="222" style="float:left; margin: 10px" alt="A screenshot of Delta Chat on desktop" />
+</a> 
+<a href="../assets/blog/screenshots/2020-01-09-delta-chat-iOS-weekend-group-chat.png">
+<img src="../assets/blog/screenshots/2020-01-09-delta-chat-iOS-weekend-group-chat-thumbnail.png" width="110" height="219" style="margin: 10px" alt="A screenshot of Delta Chat on IOS" />
+</a>
```
Comment (missytake) [inline en/index.md]:
```suggestion

<a href="../assets/blog/desktop-screenshot.png">
<img src="../assets/blog/desktop-screenshot-thumbnail.png" width="280" height="222" style="float:left; margin: 10px" alt="A screenshot of Delta Chat on desktop" />
</a> 

<a href="../assets/blog/screenshots/2020-01-09-delta-chat-iOS-weekend-group-chat.png">
<img src="../assets/blog/screenshots/2020-01-09-delta-chat-iOS-weekend-group-chat-thumbnail.png" width="110" height="219" style="margin: 10px" alt="A screenshot of Delta Chat on IOS" />
</a>
```

I think the empty lines are important, they are rendered to be `<p>` tags in the status quo. The `<p>` tags around the images are the difference.

**Review by r10s — APPROVED** [review]
lgtm, thanks a lot.

it is slightly less sharp on hi-res (retina) displays (that can make sense of additional resolution), however, the saved traffic and loading times are much more important, so it is a great tradeoff, no need for further tweaking (the screenshots need to be updated anyways)

for translations: yes, that has to go to the regular `t-dance.sh` after merging before it becomes visible on other languages.

**Comment by r10s** [comment]
> for translations: yes, that has to go to the regular t-dance.sh after merging before it becomes visible on other languages

did a pr, #708

**Pull request merged**